src/app/components/app-layout/app-layout.component.html

```diff
@@ -53,14 +53,21 @@
     </div>
 
   </header>
-
+  <!-- 
```

Comment:
Why is this commented out , should it be removed?

Reply:
During development I tried playing with different solutions to make the alert fixed. I've removed it.